glossary/moon.md

```diff
@@ -1,16 +1,33 @@
 +++
 title = "Moon"
-
 template = "doc.html"
 [extra]
 category = "arvo"
 +++
 
-A **moon** is a kind of [ship](../ship) on the [Arvo](../arvo) network. Moons are issued by planets, and are meant to be used for the issuing planet's connected devices, such as phones, desktops, smart TVs, and digital thermostats. A moon is not independent; it is always subordinated to the planet that issued it.
+A **moon** is a kind of [ship](../ship) on the [Arvo](../arvo) network. Moons
+are child identities issued by [planets](../planet), [stars](../stars), and
+[galaxies](../galaxy). A moon is not independent; it is always subordinate to
+the ship that issued it. By this we mean that the networking keys for the moon
+are controlled by the parent, and may be altered or revoked at any time by that parent.
+
+Moons may be utilized for many purposes, including but not limited to:
+ - identities for the issuing ship's connected devices, such as phones, desktops, smart TVs, and digital thermostats
+ - vanity identities
+ - backup identities incase the parent identity cannot be accessed
+ - identities for children
```

Comment:
Prolly remove this because we lack tooling for the kinds of things people would generally want before giving anyone else a moon.

Reply:
I think besides devices, the rest are fine. We see regular use of vanity identities (~master-rapfyr-diglyt`) and backup identities, and while there are no parental controls of any kind on moons, I'd still find it appropriate to give a moon to e.g. a mature teenager whose key could be revoked in case of misbehavior.

https://github.com/urbit/docs/pull/1078/commits/4abc4536a117aeeafcc4c96c1bcfafc0e88aff6a

Reply:
@baudtack ?